packages/elastic-sip-trunking/tests/rest/v1/access-control-list/access-control-lists.steps.ts

```diff
@@ -0,0 +1,324 @@
+  assert.equal(accessControlList.projectId, 'tinyfrog-jump-high-over-lilypadbasin');
+  assert.deepEqual(accessControlList.createTime, new Date('2024-06-06T14:42:42.892741384Z'));
+  assert.equal(accessControlList.updateTime, null);
+  assert.ok(accessControlList.ipRanges);
+  const ipRange = accessControlList.ipRanges[0];
+  assert.equal(ipRange.id, '01W4FFL35P4NC4K35IPRANGE01');
+  assert.equal(ipRange.accessControlListId, '01W4FFL35P4NC4K35SIPACL001');
+  assert.equal(ipRange.description, 'Location 1');
+  assert.equal(ipRange.ipAddress, '15.15.15.15');
+  assert.equal(ipRange.range, 32);
+  assert.equal(ipRange.projectId, 'tinyfrog-jump-high-over-lilypadbasin');
+  assert.deepEqual(ipRange.createTime, new Date('2024-06-06T14:42:42.896695235Z'));
+  assert.equal(ipRange.updateTime, null);
+});
+
+When('I send a request to list the existing Access Control Lists', async () => {
+  listResponse = await accessControlListsApi.list({});
+});
+
+Then('the response contains {string} Access Control Lists', (expectedAnswer: string) => {
+  const expectedAclsCount = parseInt(expectedAnswer, 10);
+  assert.equal(listResponse.data.length, expectedAclsCount);
+  accessControlList = listResponse.data[0];
+  assert.equal(accessControlList.id, '01W4FFL35P4NC4K35SIPACL001');
+  assert.deepEqual(accessControlList.createTime, new Date('2024-06-06T14:42:42Z'));
+  const ipRange = accessControlList.ipRanges[0];
+  assert.deepEqual(ipRange.createTime, new Date('2024-06-06T14:42:42Z'));
+});
+
+When('I send a request to list all the Access Control Lists', async () => {
+  aclsList = [];
+  for await (const acl of accessControlListsApi.list({})) {
+    aclsList.push(acl);
+  }
+});
+
+When('I iterate manually over the Access Control Lists pages', async () => {
+  aclsList = [];
+  listResponse = await accessControlListsApi.list({});
+  aclsList.push(...listResponse.data);
+  pagesIteration = 1;
+  let reachedEndOfPages = false;
+  while (!reachedEndOfPages) {
+    if (listResponse.hasNextPage) {
+      listResponse = await listResponse.nextPage();
+      aclsList.push(...listResponse.data);
+      pagesIteration++;
+    } else {
+      reachedEndOfPages = true;
+    }
+  }
+});
+
+Then('the Access Control Lists list contains {string} Access Control Lists',  (expectedAnswer: string) => {
+  const expectedAclsCount = parseInt(expectedAnswer, 10);
+  assert.equal(aclsList.length, expectedAclsCount);
+});
+
+Then('the Access Control Lists iteration result contains the data from {string} pages',  (expectedAnswer: string) => {
+  const expectedPagesCount = parseInt(expectedAnswer, 10);
+  assert.equal(pagesIteration, expectedPagesCount);
+});
+
+When('I send a request to retrieve an Access Control List', async () => {
+  accessControlList = await accessControlListsApi.get({
+    id: '01W4FFL35P4NC4K35SIPACL001',
+  });
+});
+
+Then('the response contains the Access Control List details', () => {
+  assert.equal(accessControlList.id, '01W4FFL35P4NC4K35SIPACL001');
```

Comment:
`name` and `ipRanges`are required fields.
May be the test could ensure they are received well.

Reply:
Fixed